[2021-05-30 19:08:23,408] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: data_engineering_assignment_2.setup_oparator 2021-05-30T19:08:20.535049+00:00 [queued]>
[2021-05-30 19:08:23,459] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: data_engineering_assignment_2.setup_oparator 2021-05-30T19:08:20.535049+00:00 [queued]>
[2021-05-30 19:08:23,464] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-30 19:08:23,465] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2021-05-30 19:08:23,467] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-30 19:08:23,491] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): setup_oparator> on 2021-05-30T19:08:20.535049+00:00
[2021-05-30 19:08:23,501] {standard_task_runner.py:52} INFO - Started process 59 to run task
[2021-05-30 19:08:23,508] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_engineering_assignment_2', 'setup_oparator', '2021-05-30T19:08:20.535049+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment_2_airflow_dags.py', '--cfg-path', '/tmp/tmp_1rdszsa', '--error-file', '/tmp/tmpakpzbnva']
[2021-05-30 19:08:23,510] {standard_task_runner.py:77} INFO - Job 11: Subtask setup_oparator
[2021-05-30 19:08:23,621] {logging_mixin.py:104} INFO - Running <TaskInstance: data_engineering_assignment_2.setup_oparator 2021-05-30T19:08:20.535049+00:00 [running]> on host d1b2a4d53792
[2021-05-30 19:08:23,732] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mohannad alsouqi
AIRFLOW_CTX_DAG_ID=data_engineering_assignment_2
AIRFLOW_CTX_TASK_ID=setup_oparator
AIRFLOW_CTX_EXECUTION_DATE=2021-05-30T19:08:20.535049+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-30T19:08:20.535049+00:00
[2021-05-30 19:08:23,734] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-30 19:08:23,744] {bash.py:158} INFO - Running command: pip install sqlalchemy matplotlib sklearn
[2021-05-30 19:08:23,804] {bash.py:169} INFO - Output:
[2021-05-30 19:08:25,048] {bash.py:173} INFO - WARNING: The directory '/home/airflow/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
[2021-05-30 19:08:25,311] {bash.py:173} INFO - Requirement already satisfied: sqlalchemy in /home/airflow/.local/lib/python3.6/site-packages (1.3.24)
[2021-05-30 19:08:25,360] {bash.py:173} INFO - Requirement already satisfied: matplotlib in /home/airflow/.local/lib/python3.6/site-packages (3.3.4)
[2021-05-30 19:08:25,894] {bash.py:173} INFO - Collecting sklearn
[2021-05-30 19:08:26,346] {bash.py:173} INFO -   Downloading sklearn-0.0.tar.gz (1.1 kB)
[2021-05-30 19:08:31,910] {bash.py:173} INFO - Requirement already satisfied: numpy>=1.15 in /home/airflow/.local/lib/python3.6/site-packages (from matplotlib) (1.19.5)
[2021-05-30 19:08:31,924] {bash.py:173} INFO - Requirement already satisfied: kiwisolver>=1.0.1 in /home/airflow/.local/lib/python3.6/site-packages (from matplotlib) (1.3.1)
[2021-05-30 19:08:31,929] {bash.py:173} INFO - Requirement already satisfied: python-dateutil>=2.1 in /home/airflow/.local/lib/python3.6/site-packages (from matplotlib) (2.8.1)
[2021-05-30 19:08:31,940] {bash.py:173} INFO - Requirement already satisfied: pillow>=6.2.0 in /home/airflow/.local/lib/python3.6/site-packages (from matplotlib) (8.2.0)
[2021-05-30 19:08:31,951] {bash.py:173} INFO - Requirement already satisfied: cycler>=0.10 in /home/airflow/.local/lib/python3.6/site-packages (from matplotlib) (0.10.0)
[2021-05-30 19:08:31,962] {bash.py:173} INFO - Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.3 in /home/airflow/.local/lib/python3.6/site-packages (from matplotlib) (2.4.7)
[2021-05-30 19:08:32,757] {bash.py:173} INFO - Collecting scikit-learn
[2021-05-30 19:08:32,838] {bash.py:173} INFO -   Downloading scikit_learn-0.24.2-cp36-cp36m-manylinux2010_x86_64.whl (22.2 MB)
[2021-05-30 19:08:37,118] {bash.py:173} INFO - Requirement already satisfied: six>=1.5 in /home/airflow/.local/lib/python3.6/site-packages (from python-dateutil>=2.1->matplotlib) (1.15.0)
[2021-05-30 19:08:37,238] {bash.py:173} INFO - Requirement already satisfied: scipy>=0.19.1 in /home/airflow/.local/lib/python3.6/site-packages (from scikit-learn->sklearn) (1.5.4)
[2021-05-30 19:08:37,535] {bash.py:173} INFO - Collecting joblib>=0.11
[2021-05-30 19:08:37,621] {bash.py:173} INFO -   Downloading joblib-1.0.1-py3-none-any.whl (303 kB)
[2021-05-30 19:08:37,941] {bash.py:173} INFO - Collecting threadpoolctl>=2.0.0
[2021-05-30 19:08:38,026] {bash.py:173} INFO -   Downloading threadpoolctl-2.1.0-py3-none-any.whl (12 kB)
[2021-05-30 19:08:38,186] {bash.py:173} INFO - Building wheels for collected packages: sklearn
[2021-05-30 19:08:38,194] {bash.py:173} INFO -   Building wheel for sklearn (setup.py): started
[2021-05-30 19:08:40,401] {bash.py:173} INFO -   Building wheel for sklearn (setup.py): finished with status 'done'
[2021-05-30 19:08:40,404] {bash.py:173} INFO -   Created wheel for sklearn: filename=sklearn-0.0-py2.py3-none-any.whl size=1316 sha256=82597dcb04416b4196aa61e5027a9f131ebf2671675967a799eaa26b3e0e6357
[2021-05-30 19:08:40,406] {bash.py:173} INFO -   Stored in directory: /tmp/pip-ephem-wheel-cache-axs_77rj/wheels/23/9d/42/5ec745cbbb17517000a53cecc49d6a865450d1f5cb16dc8a9c
[2021-05-30 19:08:40,410] {bash.py:173} INFO - Successfully built sklearn
[2021-05-30 19:08:51,695] {bash.py:173} INFO - Installing collected packages: joblib, threadpoolctl, scikit-learn, sklearn
[2021-05-30 19:08:55,932] {bash.py:173} INFO - Successfully installed joblib-1.0.1 scikit-learn-0.24.2 sklearn-0.0 threadpoolctl-2.1.0
[2021-05-30 19:08:56,592] {bash.py:173} INFO - WARNING: You are using pip version 20.2.4; however, version 21.1.2 is available.
[2021-05-30 19:08:56,593] {bash.py:173} INFO - You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
[2021-05-30 19:08:56,755] {bash.py:177} INFO - Command exited with return code 0
[2021-05-30 19:08:56,817] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=data_engineering_assignment_2, task_id=setup_oparator, execution_date=20210530T190820, start_date=20210530T190823, end_date=20210530T190856
[2021-05-30 19:08:56,885] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-05-30 19:08:56,919] {local_task_job.py:146} INFO - Task exited with return code 0
